plugins/routemaster-sentry/README.md

```diff
@@ -0,0 +1,11 @@
+### routemaster-sentry
+
+Usage, in your Routemaster configuration file:
+
+```yaml
+plugins:
+  logging:
+    - class: routemaster_sentry.logger:SentryLogger
+      kwargs:
+        dsn: https://xxxxxxx:xxxxxxx@sentry.io/xxxxxxx
```

Comment:
I realise we already have some credentials in the routemaster config file (specifically webhook API keys end up there), though I wonder whether we should look at a way to separate the configuration of the routemaster instance itself from the specification of the state machine. I'm not sure where the boundary for that is (it's certainly fuzzy at the moment), though I wonder whether this is something that will be needed if we want to be able to deploy the credentials in an explicitly secure manner.

Reply:
Yep, we will need this, but I don't think we need it in this PR.

My thinking is that the configuration is relatively complex, and dynamic per plugin, and that it's nice to have config in one place (at least per plugin), so I think a mechanism to signal in the yaml to look for an environment variable instead might be the best option.

I realise this might sound a bit odd, but I've used a similar system in the past: https://github.com/yesodweb/yesod/wiki/Configuration

Something like:

```yaml
plugin:
  logging:
    - class: routemaster_sentry.logger:SentryLogger
      kwargs:
        dsn: $ENV[SENTRY_DSN]
```

This keeps it simple for the basic use-case, and in the advanced use-case still keeps the core of the configuration in one place.

Reply:
ooh, that's quite nice